steps/runAs.dhall

```diff
@@ -0,0 +1,9 @@
+let Step = ../schemas/Step.dhall
+
+in  λ(args : { name : Text, run : Text }) →
```

Comment:
Hm, I think in this case users can do:
```
GithubActions.steps.run { run = "cmd" } // { name = "name" }
```
what are your thoughts here?

Reply:
Oh that's fair enough I guess, didn't know we could do this! :+1: (I'll remove the `RunAs` step).